package.json

```diff
@@ -22,6 +22,7 @@
     "history": "^5.0.0",
     "immer": "^8.0.1",
     "jwt-decode": "^3.1.2",
+    "mui-rte": "^1.28.0",
```

Comment:
You have installed two packages but haven't used them anywhere. If you don't need them, remove those packages.

Reply:
okay...